generators/app/templates/common/.pre-commit-config.yaml

```diff
@@ -15,27 +15,27 @@ repos:
 
 - repo: local
   hooks:
-  - id: black
-    name: Formatting (black)
-    entry: black
+  - id: ruff-format
```

Comment:
Here as well, for consistency we can use same hook ids as make commands `format-fix`, `lint-check`, `type-check`